README.md

```diff
@@ -1,2 +1,34 @@
-# opentelemetry
-Get started with our Elastic Distros of OpenTelemetry
+# Elastic OpenTelemetry Distributions
+
+[OpenTelemetry](https://opentelemetry.io/docs/) is a vendor- and tool-neutral observability framework for collecting, processing, and exporting telemetry data.
+You can send your telemetry data to Elastic Observability using OpenTelemetry the following ways:
+
+- Use the **Elastic Distribution for OpenTelemetry Collector**, an extension of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), to collect and send logs and host metrics to [Elastic Cloud](https://cloud.elastic.co/).
+- Use the **Elastic Distributions for OpenTelemetry**, extensions of select [OpenTelemetry language SDKs](https://opentelemetry.io/docs/languages/), to instrument your applications and send logs, traces, and metrics to [Elastic Cloud](https://cloud.elastic.co/). The following languages are currently supported: Java, .NET, Node.js, and Python.
+- Configure your upstream [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution for the collector to collect logs and metrics and send them to Elastic Observability.
+
+This diagram provides a quick overview on how the different components work together. Refer to the [components](docs/collector-components.md) for a more in-depth look.
+
+![Diagram of the OpenTelemetry flow](docs/images/elastic-otel-overview.png)
+
+## Get started with the Elastic Distribution for OpenTelemetry Collector
+
+These pages detail the components and how to configure the Elastic Distribution for OpenTelemetry Collector ("the collector").
```

Comment:
>("the collector")

Elastic OTel Collector for shortened requirements. "the collector" might be confusing as there are many.